frontend/src/Dashboard/HistoricData.js

```diff
@@ -24,9 +28,12 @@ import { VscDashboard } from 'react-icons/vsc';
 import { GiDrop, GiChaliceDrops, GiCarBattery } from 'react-icons/gi';
 import moment from 'moment';
 import { useParams } from 'react-router-dom';
+import Loader from './Loader.js';
 
 export default function HistoricData() {
   const [weatherData, setWeatherData] = useState(null);
+  const [station, setStation] = useState('');
+  const [stationsWithData, setStationsWithData] = useState([]);
```

Comment:
Szerintem ez is lehetne null default-on. Mi van, ha az eredmény egy üres lista az API-n? Akkor loaderezünk, mert üres a tömb?